### PR TITLE
fix a bug where the volume-group was not created

### PR DESCRIPTION
**Pull request by mwennrich**
- for ephemeral volumes, the volume-group was not created if it didn't already exist.
- removed github.com/google/lvmd/commands dependency

### CODE REVIEW

File: pkg/lvm/lvm.go [hunk 1]
```diff
@@ -476,7 +481,7 @@ func CreateVG(name string, devicesPattern []string) (string, error) {
 		return name, nil
 	}
 
-	physicalVolumes, err := devices(devicesPattern)
+	physicalVolumes, err := devices(dp)
```
Comment (majst01) [inline pkg/lvm/lvm.go:484]: i think you should trim leading and trailing whitespaces from every devicePattern

Comment (mwennrich) [reply pkg/lvm/lvm.go:477]: https://github.com/metal-stack/csi-driver-lvm/pull/19/files#diff-7c238bacac13d6e6eb36c3fa42a6f44dR447

Comment (majst01) [reply pkg/lvm/lvm.go:477]: OK, but i meant the resulting parts will also have whitespaces around if for example someone specifies:

```
devicesPattern := " nvme* ,  sda* "
```

File: go.mod [hunk 2]
```diff
@@ -7,7 +7,6 @@ require (
 	github.com/docker/go-units v0.4.0
 	github.com/golang/protobuf v1.3.3 // indirect
 	github.com/google/gofuzz v1.1.0 // indirect
-	github.com/google/lvmd v0.0.0-20190916151813-e6e28ff087f6
```
Comment (majst01) [inline go.mod:10]: This is great, we should do the same in the old driver as well, WDYT ?

File: pkg/lvm/lvm.go (hunk 1 above)
Comment (mwennrich) [reply pkg/lvm/lvm.go:477]: The code path basically is:

```
	dp := strings.Split(devicesPattern, ",")
	for _, devicePattern := range dp  {
		matches, err := filepath.Glob(strings.TrimSpace(devicePattern))

```
should IMHO be ok so?

**Pull request merged**